wingtips-core/src/main/java/com/nike/wingtips/TraceAndSpanIdGenerator.java

```diff
@@ -8,9 +8,13 @@
 import java.util.Random;
 
 /**
- * ID generation class for use with trace IDs, span IDs, and parent span IDs for the {@link Span} class. Call the static {@link #generateId()} method whenever you need to create
- * a new ID. These IDs are returned as strings but they can be parsed by {@link Long#parseLong(String)} into long values. We use longs to conform to the Google Dapper paper and
- * Twitter ZipKin distributed tracing implementation.
+ * ID generation class for use with trace IDs, span IDs, and parent span IDs for the {@link Span} class. Call the static {@link #generateId()} method whenever you
+ * need to create a new ID. These IDs are fundamentally 64-bit longs, but they are returned by {@link #generateId()} encoded as unsigned and in hexadecimal format.
```

Comment:
not worth changing, but I tend to think of them as 8 opaque bytes packed into a long. Helps me remember to not treat ids numerically (eg expect order etc to make sense)